BlogTemplate/Models/BlogDataStore.cs

```diff
@@ -20,13 +20,44 @@ public BlogDataStore(IFileSystem fileSystem)
         {
             _fileSystem = fileSystem;
             InitStorageFolder();
+            InitCurrentId();
         }
         public void InitStorageFolder()
         {
             _fileSystem.CreateDirectory(StorageFolder);
             _fileSystem.CreateDirectory(DraftsFolder);
         }
 
+        public void InitCurrentId()

```

Comment:
Method 1: because this is a behavior called only by the constructor, it's really a behavior *of* the constructor.  It just happens to be refactored into a private method.  So you can call the constructor (passing in a fake file system of your choice) and verify it gets the right results.  In order to find the value of `CurrentId` you may need to save a new post (since that's the only code which reads the value).

Method 2 would be to make this method internal, and make sure that this project gives internal access to the test project.